modules/tinymce/src/themes/silver/main/ts/Autocompleter.ts

```diff
@@ -214,7 +214,9 @@ const register = (editor: Editor, sharedBackstage: UiFactoryBackstageShared) =>
     getView: () => InlineView.getContent(autocompleter)
   };
 
-  AutocompleterEditorEvents.setup(autocompleterUiApi, editor);
+  if (editor.hasPlugin('rtc') === false) {
```

Comment:
Any reason to not simplify this to `!editor.hasPlugin('rtc')`?

Reply:
Just thing === false is more explicit ! is sometimes hard to miss and you thing it's a truth check but I can change it if that is the style you prefer.

Reply:
Fair enough, it should get minified down to the shorthand anyways, so I'm fine with this.